api/chaos/experimentation/v1/experimentation.proto

```diff
@@ -61,7 +67,10 @@ message CancelExperimentRunRequest {
     pattern : "{id}"
   };
 
-  string id = 1;
+  // The identifier of an experiment run to cancel.
+  string id = 1 [ (validate.rules).string = {min_bytes : 1} ];
+  // The reason for the cancelation request.
+  string reason = 2 [ (validate.rules).string = {min_bytes : 1, max_len : 32} ];
```

Comment:
Thoughts on bumping the reason limit?

Reply:
I can bump it 👍  - I did not want to introduce a database migration in my PR but it blew up anyway so I may as well introduce it here.

Reply:
increased to 100